### PR TITLE
Add blog post for 4.4 dev 3

### PR DESCRIPTION
**Pull request by clayjohn**
~Still missing cover photo~

TL;DR: for socials
- 330 commits packed with features and enhancements
- Faster rendering and improvements for PS1-style graphics
- Faster editor startup and filesystem operations
- Animation markers to allow more customization and flexibility in existing animations
- Tool buttons to enhance the power of tool scripts and editor tools

Showcased game:
- Fogpiercer by Mad Cookies Studio
- https://store.steampowered.com/app/3219010/Fogpiercer/
- https://x.com/MadCookiesGames
- Screenshot: https://shared.akamai.steamstatic.com/store_item_assets/steam/apps/3219010/ss_3d5c4baed1459bb4e7777818da2481911f93dfec.jpg?t=1726829136

### CODE REVIEW

File: collections/_article/dev-snapshot-godot-4-4-dev-3.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+combined result is a lot of great stuff compressed into a short amount of time.
+
+Many of the changes in this release are bug fixes that will be backported to Godot 4.3 and released in 4.3.1! So please
+test this release well so we can be confident with the changes and release 4.3.1 with them as soon as possible.
+
+Keep in mind that while we try to make sure each dev snapshot is stable enough for general testing, this is by
+definition a pre-release piece of software. Be sure to make frequent backups, or use a version control system such as
+Git, to preserve your projects in case of corruption or data loss.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.4.dev2/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
```
Comment (m4gr3d) [inline collections/_article/dev-snapshot-godot-4-4-dev-3.md]: Can we add a link for the XR editor here as well. e.g:

You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.4.dev2/), [**XR editor**](https://www.meta.com/experiences/godot-game-engine/7713660705416473/) or the **Android editor** for this release.

Comment (akien-mga) [reply collections/_article/dev-snapshot-godot-4-4-dev-3.md]: We can, though this whole paragraph will need rethinking eventually as it's a very lengthy blurb and I'm not sure how impactful it is. It might be better to feature all these in the download card directly.

BTW, how up to date are we on the XR editor and Android editor? Do you still manage to push new releases shortly after a dev snapshot is built? I've never figured out how to actually get the pre-release version of the Android editor from Google Play, so I feel we might want some dedicated instructions for this in the docs instead of bringing it along in each blog post.

Comment (m4gr3d) [reply collections/_article/dev-snapshot-godot-4-4-dev-3.md]: > BTW, how up to date are we on the XR editor and Android editor? Do you still manage to push new releases shortly after a dev snapshot is built?

Both versions are kept up-to-date, although for the XR editor, the dev releases are pushed directly to production given the user base is much smaller, whereas for the Android editor, the dev releases are pushed to internal and opt-in testing.

> I've never figured out how to actually get the pre-release version of the Android editor from Google Play, so I feel we might want some dedicated instructions for this in the docs instead of bringing it along in each blog post.

We have a page in the docs for the Android editor, where we could provide instructions for how to access the dev builds - https://docs.godotengine.org/en/stable/tutorials/editor/using_the_android_editor.html#limitations-known-issues
I'm planning to add a similar page for the XR editor.

Comment (akien-mga) [reply collections/_article/dev-snapshot-godot-4-4-dev-3.md]: > I've never figured out how to actually get the pre-release version of the Android editor from Google Play, 

Well seems like I finally managed, looks like I had subscribed to the group with the wrong google account :P

**Comment by akien-mga** [comment]
Added cover image, some more details, converted images to webp, and version metadata for the download links.

**Review by akien-mga — APPROVED** [review]

File: collections/_article/dev-snapshot-godot-4-4-dev-3.md (hunk 1 above)
Comment (akien-mga) [reply collections/_article/dev-snapshot-godot-4-4-dev-3.md]: Done!

**Comment by akien-mga** [comment]
Thanks!
